third-culture-software/bhima
# Removed condition from shipment items

Removed shipment_id from shipment items.  The condition of items will be tracked via asset scans.

Also fixed an issue in the shipment.json files that was preventing the shipment status options from being translated in Shipments registry Search function.

This PR also fixed some issues with previous merged PRs that were preventing migrations from working properly.

Closes https://github.com/IMA-WorldHealth/bhima/issues/6497

**TESTING**
- Create a shipment.  Notice that the condition is no longer needed for inventory added to the shipment.
- Exercise the full cycle
  - Create  new shipment (from principle depot to secondary)
  - Add item(s)
  - Submit
  - Go to Shipments Registry and mark the shipment as ready to go.
  - Do stock exit (Depot --> Select Secondary depot) for the shipment items
  - Do Stock entry (change depot to secondary depot) from shipment
- Verify the operations above
